gohike-ui/src/components/FindHikes/SideBar.jsx

```diff
@@ -171,6 +177,21 @@ export function HikePopout({ selectedHike, setSelectedHike }) {
                     `Trail Condition: ${selectedHike?.conditionStatus}`}
                 </span>
                 <p>{`"${selectedHike?.summary}"`}</p> 
+                <div className="comment-section">
+                    <p>Comments</p>
+                    <div className="comments"></div>
+                </div>
+                <div className="comment-form">
+                    <input 
+                            className="comment-input" 
```

Comment:
@laurenkiyomi  Are these formatted correctly? May take a look at some JS formatting tool to help autoformat your code?